### DISCUSSION TITLE
Using Solito with a bare workflow non-Expo React Native app?

### DISCUSSION THREAD

**Discussion by williamlabanowski** [nandorojo/solito#212] [Q&A] [answered]
Are there any limitations with using Solito for a bare workflow non-Expo React Native app? I ejected my Expo integration a while back and am looking to create a web-based version of my React Native app with server-side rendering via NextJS. What you've built here is fantastic but it seems it's quite geared towards usage with Expo. However, I see no reason why it wouldn't be just as useful for a bare workflow project unless I'm missing something?

**Comment by nandorojo** [comment, marked as answer]
shouldn’t matter. i prefer the expo + dev client flow though.

**Reply by williamlabanowski** [reply]
Ok that's what I figured. Thanks a ton!
